features/FEATURE_BLE/ble/pal/GapTypes.h

```diff
@@ -446,6 +446,9 @@ struct advertising_event_properties_t {
                 break;
             case advertising_type_t::ADV_NONCONN_IND:
                 break;
+            case advertising_type_t::ADV_NONSCAN_IND:
+                connectable = true;
```

Comment:
`use_legacy_pdu` should be set too false.

Reply:
I assumed it was, seeing as only a few lines above there's a:
```
 use_legacy_pdu = true;
```
but apparently the default is already true

Reply:
Actually the default constructor does set it to false and either way it's irrelevant since the type shouldn't set it as it's set by an explicit passed in parameter.

Reply:
```
    pal::advertising_event_properties_t event_properties(params.getType());
    event_properties.use_legacy_pdu = params.getUseLegacyPDU();
```

Reply:
We discussed this and the changes required to make this neat are outside of scope for this PR and functionally the code is correct.